<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="6" hostname="user-PC1" name="com.tranquil.testcases.QuotationSave" tests="7" failures="0" timestamp="2020-01-10T14:40:18 IST" time="16.545" errors="1">
  <testcase name="quotationreport_data_writing" time="16.545" classname="com.tranquil.testcases.QuotationSave">
    <error type="java.lang.NoClassDefFoundError" message="org/apache/poi/hssf/usermodel/HSSFWorkbook">
      <![CDATA[java.lang.NoClassDefFoundError: org/apache/poi/hssf/usermodel/HSSFWorkbook
at Helper.writeExcel.<init>(writeExcel.java:26)
at com.tranquil.testcases.QuotationSave.quotationreport_data_writing(QuotationSave.java:499)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: org.apache.poi.hssf.usermodel.HSSFWorkbook
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 30 more
]]>
    </error>
  </testcase> <!-- quotationreport_data_writing -->
  <testcase name="verify_quotation" classname="com.tranquil.testcases.QuotationSave">
    <skipped/>
  </testcase> <!-- verify_quotation -->
  <testcase name="save_quotition" classname="com.tranquil.testcases.QuotationSave">
    <skipped/>
  </testcase> <!-- save_quotition -->
  <testcase name="approve_quotation" classname="com.tranquil.testcases.QuotationSave">
    <skipped/>
  </testcase> <!-- approve_quotation -->
  <testcase name="edit_quotation" classname="com.tranquil.testcases.QuotationSave">
    <skipped/>
  </testcase> <!-- edit_quotation -->
  <testcase name="afterapprove_quotationview" classname="com.tranquil.testcases.QuotationSave">
    <skipped/>
  </testcase> <!-- afterapprove_quotationview -->
  <testcase name="view_quotation" classname="com.tranquil.testcases.QuotationSave">
    <skipped/>
  </testcase> <!-- view_quotation -->
</testsuite> <!-- com.tranquil.testcases.QuotationSave -->
